jupyterhub/auth.py

```diff
@@ -123,17 +127,32 @@ class LocalAuthenticator(Authenticator):
         should I try to create the system user?
         """
     )
-    system_user_home = Unicode('', config=True,
-        help="""Specify root home directory for users if different from system default.
+    add_user_cmd = Command(config=True,
+        help="""add user command
         
-        Passed to `useradd -b`.
-        If specified, when system users are created their home directories will be created in
+        USERNAME will be replaced with the user's name.
```

Comment:
For each element in the list, the string `USERNAME` will be replaced with the user's username. The username will also be appended as the final argument.